datasetUtils/checkForNewVersion.go

```diff
@@ -49,19 +62,19 @@ func CheckForNewVersion(client *http.Client, APP string, VERSION string, interac
 		log.Fatalf("Illegal version number:%v", VERSION)
 	}
 	log.Printf("Latest version: %s", latestVersion)
-	if majorLatest > majorCurrent {
-		log.Println("You must upgrade to a newer version")
-		log.Println("Current Version: ", VERSION)
-		log.Println("Latest  Version: ", latestVersion)
-		log.Println("Use the following command to update:")
-		log.Println("curl -O " + DeployLocation + APP + ";chmod +x " + APP)
-		log.Fatalf("Execution stopped, please update program now.\n")
-	} else if version.Compare(latestVersion, VERSION, ">") {
+	
+	// Get the operating system name
+	osName := runtime.GOOS
+	
+	// Generate the download URL
+	downloadURL := fmt.Sprintf("%s/scicat-cli_%s_%s_x86_64.tar.gz", DeployLocation, latestVersion, strings.Title(osName))
```

Comment:
does this work? when I copy the link from the release I see: 
https://github.com/paulscherrerinstitute/scicat-cli/releases/download/v0.1.0/scicat-cli_.0.1.0_Linux_x86_64.tar.gz

(notice the `tag` after /download/)

Reply:
You're right, I'll fix it.